.github/workflows/phgv2_ci.yml

```diff
@@ -0,0 +1,35 @@
+name: PHGv2 CI
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - opened
+    branches:
+      - main
```

Comment:
Should we add the path part we needed for the CD branch here as well so it only triggers to source code changes?  Might be good for now, but if we include docs might need to add more to it as we may want a trigger off documentation changes as well.

Reply:
This has now been added

Reply:
Does this mean "opening a pull request against main"?

Reply:
@tcasstevens - I copied this from the BioKotlin repo. That is my interpretation as well